core/orchestrator.go

```diff
@@ -321,14 +331,28 @@ func (orch *orchestrator) acceptablePrice(sender ethcommon.Address, ep *net.Pric
 	return nil
 }
 
-func NewOrchestrator(n *LivepeerNode) *orchestrator {
+func (orch *orchestrator) isActive() (bool, error) {
+	filter := &common.DBOrchFilter{
+		CurrentRound: orch.rm.LastInitializedRound(),
+		Addresses:    []ethcommon.Address{orch.Address()},
+	}
+	orchs, err := orch.node.Database.SelectOrchs(filter)
```

Comment:
When running in orchestrator mode, the DB's orchestrator table is currently not populated so this query will always return an empty slice.

Reply:
added populating DB to the `setupOrchestrator` helper in livepeer.go , which doesn't seem to do much currently except for logging whether the orchestrator is active. 

the fixup just moves around that helper call to avoid doing those RPC calls when running `./test_args.sh`